modules/stanford_soe_helper_magazine/stanford_soe_helper_magazine.views_default.inc

```diff
@@ -10,6 +10,389 @@
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_3']['ui_name'] = 'Content: Issue Accent color';
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_3']['label'] = '';
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_3']['exclude'] = TRUE;
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_3']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_3']['type'] = 'taxonomy_term_reference_plain';
+  /* Field: Content: Accent color */
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['id'] = 'field_s_mag_issue_accent_color_4';
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['table'] = 'field_data_field_s_mag_issue_accent_color';
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['field'] = 'field_s_mag_issue_accent_color';
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['relationship'] = 'reverse_field_s_mag_issue_article_5_node';
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['label'] = '';
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['exclude'] = TRUE;
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['field_s_mag_issue_accent_color_4']['type'] = 'taxonomy_term_reference_plain';
+  /* Field: Content: Edit link */
+  $handler->display->display_options['fields']['edit_node']['id'] = 'edit_node';
+  $handler->display->display_options['fields']['edit_node']['table'] = 'views_entity_node';
+  $handler->display->display_options['fields']['edit_node']['field'] = 'edit_node';
+  $handler->display->display_options['fields']['edit_node']['label'] = '';
+  $handler->display->display_options['fields']['edit_node']['exclude'] = TRUE;
+  $handler->display->display_options['fields']['edit_node']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['edit_node']['text'] = 'Edit';
+  /* Field: Global: Custom text */
+  $handler->display->display_options['fields']['nothing']['id'] = 'nothing';
+  $handler->display->display_options['fields']['nothing']['table'] = 'views';
+  $handler->display->display_options['fields']['nothing']['field'] = 'nothing';
+  $handler->display->display_options['fields']['nothing']['label'] = '';
+  $handler->display->display_options['fields']['nothing']['alter']['text'] = '<div class="mag-article-container">
+  <div class="mag-article-img">[field_s_mag_article_image]</div>
+    <div class="mag-article-content-container">
+      <div class="mag-article-date">Research & Ideas – [field_s_mag_article_date]</div>
```

Comment:
You might want to use `&amp;` instead of `&` here.

Reply:
TIL! Thanks Shea!

Reply:
Actually it didn't work. It came out as &amp

![screen shot 2018-03-16 at 11 43 02](https://user-images.githubusercontent.com/284440/37538914-5c61b68c-290f-11e8-8f5c-2660f76fd0f0.png)

Reply:
Ah, ok. Wasn't sure if the raw output was html encoded or not. Appears to be. You can ignore the first comment.